pkg/network/config/config_linux.go

```diff
@@ -37,10 +40,12 @@ func (c *Config) EnabledProbes(runtimeTracer bool) (map[probes.ProbeName]struct{
 			enabled[probes.TCPRetransmit] = struct{}{}
 		}
 
-		enabled[probes.SockFDLookup] = struct{}{}
-		enabled[probes.SockFDLookupRet] = struct{}{}
-		enabled[probes.DoSendfile] = struct{}{}
-		enabled[probes.DoSendfileRet] = struct{}{}
+		if matches, _ := ebpf.VerifyKernelFuncs(filepath.Join(c.ProcRoot, "kallsyms"), []string{"sockfd_lookup_light"}); len(matches) > 0 {
```

Comment:
This condition is backwards. The function returns the *missing* functions.

Reply:
argh. good catch 👍